Automattic/wp-calypso
# Sidebar: Replace Gridicons and Material icons with WordPress icons

Part of https://github.com/Automattic/dotcom-forge/issues/9083

## Proposed Changes

Replaces sidebar icons of the Gridicons and Matericon icons libraries of the `/sites` and `/me` sections with icons from the WordPress icons library

Before | After
--- | ---
<img width="269" alt="Screenshot 2024-09-26 at 13 26 40" src="https://github.com/user-attachments/assets/ade3de85-0b8d-4258-8c54-951f0ffe18ce"> | <img width="273" alt="Screenshot 2024-09-26 at 13 26 09" src="https://github.com/user-attachments/assets/2db8f612-fdb4-446b-ad71-0f8c71db53e8">
<img width="265" alt="Screenshot 2024-10-01 at 12 55 23" src="https://github.com/user-attachments/assets/11ec7f14-df94-4e16-8513-6a68ef1c3622"> | <img width="269" alt="Screenshot 2024-10-01 at 12 56 58" src="https://github.com/user-attachments/assets/02b13ac5-e209-4bce-b43a-9989d35e07c3">




## Why are these changes being made?

To reduce the number of non-WordPress icons that we currently have

## Testing Instructions

- Use the Calypso live link below
- Go to `/sites`
- Make sure the sidebar icons look as expected (only the "Domains" icon has changed)
- Go to `/me`
- Make sure the sidebar icons look as expected (all icons except "Apps" have changed)

## Pre-merge Checklist
